Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM openjdk:8-jdk-alpine
 WORKDIR /
-COPY target/conter-0.0.1-SNAPSHOT.jar app.jar
+COPY conter-0.0.1-SNAPSHOT.jar app.jar
+COPY script.sh script.sh
 EXPOSE 8080
-ENTRYPOINT [ "sh", "-c", "java -Dutopia.search.API=$1 -Dutopia.booking.API=$2 -Dutopia.cancellation.API=$3 -Djava.security.egd=file:/dev/./urandom -jar /app.jar" ]
+ENTRYPOINT [ "sh","./script.sh" ]
```

Comment:
If it has the proper permissions, it *should* work to just call the script directly instead of specifying `sh` as its interpreter. (This form would break if someone, not thinking about how the Docker image was built, modified the script to use a Bash-specific feature and only changed the script's shebang line to reflect that.)